docs/building/cmake.rst

```diff
@@ -0,0 +1,75 @@
+.. _build_cmake:
+
+==========
+cmake
+==========
+
+Overview
+********
+This project uses CMakePresets to ensure an easy build.  Find out more on Cmake Presets here:
+https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html
+
+Prerequsites
+---------
```

Comment:
This decorations should have the same length as the tilte

Reply:
so like this?

Reply:
Almost :slightly_smiling_face: 

Let me show!